lib/PlaceLookup.php

```diff
@@ -421,6 +424,22 @@ public function lookup($aResults, $iMinRank = 0, $iMaxRank = 30)
 
         foreach ($aPlaces as &$aPlace) {
             $aPlace['importance'] = (float) $aPlace['importance'];
+
+            $licence_copyright_content = $this->oDB->getRow(
+                "SELECT extratags -> 'data_licence' AS licence, extratags -> 'data_copyright' AS copyright FROM placex where place_id = :placeid",
+                array(':placeid' => $aPlace['place_id'])
+            );
```

Comment:
You should enable returning extratags per default ($this->bExtratags) and then extract the information from there instead of running another query on the database.